src/test/java/org/gridsuite/actions/server/ContingencyListControllerTest.java

```diff
@@ -208,14 +206,14 @@ public void test() throws Exception {
                 .contentType(APPLICATION_JSON))
                 .andExpect(status().isOk()).andReturn().getResponse().getContentAsString();
 
-        UUID ticId = objectMapper.readValue(res, FormContingencyList.class).getId();
+        UUID ticId = objectMapper.readValue(res, FormContingencyList.class).getMetadata().getId();
 
         // check first form insert
         mvc.perform(get("/" + VERSION + "/form-contingency-lists")
                         .contentType(APPLICATION_JSON))
                 .andExpect(status().isOk())
                 .andExpect(content().contentTypeCompatibleWith(APPLICATION_JSON))
-                .andExpect(content().json("[{\"equipmentType\":\"GENERATOR\",\"nominalVoltage1\":{\"type\":\"GREATER_THAN\",\"value1\":100.0,\"value2\":null},\"nominalVoltage2\":null,\"countries1\":[\"BE\",\"FR\"],\"countries2\":[],\"type\":\"FORM\"}]", false));
+                .andExpect(content().json("[{\"equipmentType\":\"GENERATOR\",\"nominalVoltage1\":{\"type\":\"GREATER_THAN\",\"value1\":100.0,\"value2\":null},\"nominalVoltage2\":null,\"countries1\":[\"BE\",\"FR\"],\"countries2\":[], \"metadata\":{\"type\":\"FORM\"}}]", false));
```

Comment:
this JSON form seems bad we don't need to have a metadata level I think...

Reply:
Actually, the metadata and the other data are required for different purposes in the front. So it's quite relevant to separate them